### PR TITLE
Listing of Requested Books

### PR DESCRIPTION
**Pull request by Rownaknira**

### CODE REVIEW

File: app/views/requests/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+        <th></th>
+      </tr>
+    </thead>
+    <% @requests.each do |request|%>
+      <tr>
+        <td><%= request.title %></td>
+        <td><%= request.author %></td>
+        <td><%= request.description %></td>
+        <td><%= request.link %></td>
+        <td><%= link_to t ".cancel",request_path(request),method: :delete,data:
```
Comment (abdullahallmehedi) [inline app/views/requests/index.html.erb]: spacing

File: config/locales/en.yml [hunk 2]
```diff
@@ -1,24 +1,31 @@
...
-  framgia: "Framgia Bangladesh Ltd."
-  admin: "Welcome Admin"
-  user: "Welcome User"
+  about: About
+  admin: Welcome Admin
+  books: Books
+  contact: Contact
+  english: English
+  error: error
+  form_contains: 'The form contains '
```
Comment (abdullahallmehedi) [inline config/locales/en.yml]: ""

**Comment by abdullahallmehedi** [comment]
ok

**Comment by SSalekin** [comment]
ok

**Comment by noor-framgia** [comment]
ok

File: app/controllers/requests_controller.rb [hunk 3]
```diff
@@ -1,4 +1,9 @@
 class RequestsController < ApplicationController
+  before_action :authenticate_user!
+  def index
+    @requests = current_user.requests.all
```
Comment (dieunb) [inline app/controllers/requests_controller.rb]: no need `all`

**Pull request merged**